packages/polar/test/fixture-projects/compile-task-project/multiproject/contracts/contract_1/examples/schema.rs

```diff
@@ -0,0 +1,20 @@
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+use cosmwasm_schema::{export_schema, remove_schemas, schema_for};
+
+use sample_project::msg::{CountResponse, HandleMsg, InitMsg, QueryMsg};
+use sample_project::state::State;
+
+fn main() {
+    let mut out_dir = current_dir().unwrap();
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema(&schema_for!(InitMsg), &out_dir);
+    export_schema(&schema_for!(HandleMsg), &out_dir);
+    export_schema(&schema_for!(QueryMsg), &out_dir);
+    export_schema(&schema_for!(State), &out_dir);
+    export_schema(&schema_for!(CountResponse), &out_dir);
+}
```

Comment:
why do we need this file 3 times ? could you please check if three fixture-projects aren't exactly same.
i think two of them are same.